pymongo/collection.py

```diff
@@ -2553,7 +2553,7 @@ def watch(
             pipeline stages are valid after a ``$changeStream`` stage, see the
             MongoDB documentation on change streams for the supported stages.
           - `full_document` (optional): The fullDocument to pass as an option
-            to the ``$changeStream`` stage. Allowed values: 'updateLookup'.
+            to the ``$changeStream`` stage. Allowed values: 'updateLookup', 'whenAvailable', 'required'.
             When set to 'updateLookup', the change notification for partial
             updates will include both a delta describing the changes to the
             document, as well as a copy of the entire document that was
```

Comment:
For full_document_before_change instead of backticks:
```
`whenAvailable` and `required`
```
single quotes will render more like string literals in the docs:
```
'whenAvailable' and 'required'
```